tests/integration_tests/dashboards/security/security_rbac_tests.py

```diff
@@ -191,7 +190,7 @@ def test_get_dashboard_view__user_access_with_dashboard_permission(self):
 
         request_payload = get_query_context("birth_names")
         rv = self.post_assert_metric(CHART_DATA_URI, request_payload, "data")
-        self.assertEqual(rv.status_code, 200)
+        self.assertEqual(rv.status_code, 403)
```

Comment:
This test (which was failing after the change) indicates that either i) RBAC were supposed to expend beyond the scope/context of the dashboard, or ii) was a mistake.